### PR TITLE
#793 Tower of Hanoi Algorithm C#

### PR DESCRIPTION
**Pull request by imPGupta**
### Have you read the [Contributing Guidelines on Pull Requests]
Yes

### Description

Tower of Hanoi problem

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#793

### CODE REVIEW

**Comment by imPGupta** [comment]
@atarax665  please review my pull request.

**Review by atarax665 — APPROVED** [review]
Looks good to me. :)

**Review by iamrajiv — APPROVED** [review]

**Comment by imPGupta** [comment]
@Kajol-Kumari , Please review my PR

File: C-Sharp/math/tower_of_hanoi.cs [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+Move disk 4 from rod 1 to rod 3
+Move disk 1 from rod 2 to rod 3
+Move disk 2 from rod 2 to rod 1
+Move disk 1 from rod 3 to rod 1
+Move disk 3 from rod 2 to rod 3
+Move disk 1 from rod 1 to rod 2
+Move disk 2 from rod 1 to rod 3
+Move disk 1 from rod 2 to rod 3
+
+*/
```
Comment (Kajol-Kumari) [inline C-Sharp/math/tower_of_hanoi.cs]: Please leave one extra blank line at the end.

**Review by Kajol-Kumari — APPROVED** [review]
@imPGupta please resolve the conflict and do the minor change requested. Rest LGTM :+1:

**Comment by iamrajiv** [comment]
@all-contributors please add @imPGupta for code

**Pull request merged**